### PR TITLE
CONCF-525 | Create 1024px breakpoint

### PR DESCRIPTION
**Pull request by vforge**
Ping @bognix @Warkot @hakubo

### CODE REVIEW

File: skins/oasis/css/core/breakpoints-layout.scss [hunk 1]
```diff
@@ -219,4 +217,44 @@
...
+
+	// "special case" for 1024px tablets (think landscape iPad)
+	@media #{$breakpoint-small-plus} {
+		.WikiaPage {
+			padding: 0 $padding-width-small-plus;
+			width: $page-width-small-plus;
+		}
+
+		.wikia-main-page-tag-lcs, .main-page-tag-lcs-exploded {
+			@include width-calc("100% - 320px");
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-layout.scss]: from where is the 320px value? it's right rail width? If so please add comment

File: skins/oasis/css/core/breakpoints-layout.scss [hunk 2]
```diff
@@ -219,4 +217,44 @@
...
+		}
+
+		.main-page-tag-rcs {
+			width: 300px;
+		}
+
+		.WikiHeader {
+			min-height: 69px;
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-layout.scss]: is this value not set before somewhere? if not, why do we need to set it here?

Comment (vforge) [inline skins/oasis/css/core/breakpoints-layout.scss]: It's being overriden by small breakpoint, so we want to change it.

**Comment by vforge** [comment]
This branch is no longer needed - it's merged with CONCF-477 into CONCF-477-525.

Deleting this one.

**Pull request closed without merging**